test/Constraints/rdar38535743.swift

```diff
@@ -0,0 +1,14 @@
+// RUN: %target-typecheck-verify-swift
+// REQUIRES: objc_interop
+
+import Foundation
```

Comment:
This should use the mock SDK. Grep for 'mock-sdk' e.g. in `tests/Constraints/*_objc.swift`.

Reply:
Done!